### PR TITLE
Making top navigation fixed

### LINKED ISSUE #1137: Make the top navigation "fixed"
Thank you for contributing to this project!

### Describe your problem or feature you'd like added
The top navigation bar should remain on the page as the page is scrolled down. Users may want to interact with this at any time and we may want to add more content in here.

### Describe the solution you'd like
This may be similar to #1113 and maybe have some fixed position.

### PR DESCRIPTION
**Pull request by jonespm**
Fixes #1137
Screen_Recording_2020-12-08_at_4 09 10_PM

### CODE REVIEW

**Comment by jonespm** [comment]
@pushyamig Not sure if you were doing anything with this in #1119.

File: assets/src/containers/DashboardAppBar.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ function DashboardAppBar (props) {
 
   return (
     <div>
-      <AppBar className={classes.root} position='static'>
+      <AppBar className={classes.root} position='fixed'>
```
Comment (pushyamig) [inline assets/src/containers/DashboardAppBar.js:54]: This change is chopping off the Title of all the views like I don't see `Resource access` title at all

**Comment by jonespm** [comment]
Yeah, hmm, yeah I see that, that's not so good. ;)

File: assets/src/containers/DashboardAppBar.js (hunk 1 above)
Comment (pushyamig) [reply assets/src/containers/DashboardAppBar.js:54]: May be try adding a margin as suggested here https://stackoverflow.com/a/43795590/724238

**Comment by jonespm** [comment]
Looks like [adding an empty Toolbar will bump down the content. ](https://github.com/mui-org/material-ui/issues/16844#issuecomment-517208151)

Looking better for me now, will update the screenshot

**Review by pushyamig — APPROVED** [review]
Looks Good!

**Pull request merged**